common/motion_utils/include/motion_utils/trajectory/trajectory.hpp

```diff
@@ -1238,6 +1450,15 @@ double calcSignedArcLength(
   return signed_length_on_traj - signed_length_src_offset;
 }
 
+/**
+ * @brief calculate length of 2D distance between two points, specificed by start piont index in
+ * points contrainer and end point and its segment index in points container
```

Comment:
In some parts, the indent is wrong.

Reply:
That is fixed in my new commit.
Thanks Mr. @takayuki5168 for reviewing it.